src/components/filters/CallHistoryFilter.jsx

```diff
@@ -208,7 +208,7 @@ export default class CallHistoryFilter extends FilterBase {
                 break;
             case 'between':
                 after = today;
-                before = (30).daysAfter(today).format('{yyyy}-{MM}-{dd}');
+                before = (new Date()).advance({ days: 30}).format('{yyyy}-{MM}-{dd}');
```

Comment:
```suggestion
                before = (new Date()).advance({ days: 30 }).format('{yyyy}-{MM}-{dd}');
```

Balance the whitespace in { and }, in some way. I like "one space before and after content.

Reply:
Eh, `advance` - I don't know where that API came from, so here's something only using the Date object:

```js
let inOneMonth = new Date();
inOneMonth.setDate(inOneMonth.getDate() + 30);
```

Reply:
![image](https://user-images.githubusercontent.com/211/53698940-4706a880-3de3-11e9-8a24-7d1e786c06fd.png)